modP2pImpl/src/org/aion/p2p/impl1/tasks/TaskConnectPeers.java

```diff
@@ -158,10 +158,9 @@ public void run() {
                 } catch (Exception e) {
                     if (p2pLOG.isDebugEnabled()) {
                         p2pLOG.debug(
-                                "connect-outbound exception -> id={} ip={} reason={}",
+                                "connect-outbound exception -> id={} ip={}",
                                 node.getIdShort(),
-                                node.getIpStr(),
-                                e.toString());
+                                node.getIpStr(), e);
```

Comment:
I think that the message needs to be concatenated here, otherwise, the stack trace won't be printed for the exception